### DISCUSSION TITLE
Do I have ability to implement dragging/cancel of the limit orders (price lines)?

### DISCUSSION THREAD

**Discussion by zouxuoz** [tradingview/lightweight-charts#817] [Q&A]
Hi 👋 

Do I have ability to implement dragging/cancel of the limit orders (price lines)? Like in broker integration app
How I can add buttons to the price line?

Or you use custom version of this lib for the broker integration app?

![image](https://user-images.githubusercontent.com/9217163/128484924-b197d1ea-3b63-4356-ba46-17aff46f84ff.png)

**Comment by timocov** [comment]
`lightweight-charts` doesn't support this kind of things and it wasn't designed to be a replacement for charting_library product. Probably after https://github.com/tradingview/lightweight-charts/issues/278 and https://github.com/tradingview/lightweight-charts/issues/408 you'll be able to do so, but not out of the box.
